### PR TITLE
added README

### PR DESCRIPTION
**Pull request by Ilya-brinchuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+This project implements the online store model. It has authorization with two main roles: **User** and **Administrator**
++ Users can: login/sign in, view a list of all movies and movie session, buy  tickets for available sessions,
+  view order history
++ Administrators can: login,  view  all customers by their email, add and delete movies, sessions, cinema halls
+
+USED TECHNOLOGIES:
+======
+1. Java 11
+2. Maven
+3. Hibernate MySQL
```
Comment (Alekearl) [inline README.md]:
```suggestion
3. Hibernate, MySQL
```

File: README.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+4. Spring Core, MVC, Security
+5. Apache Tomcat
+
+TO START APP:
+======
+1. Configure Apache Tomcat
+2. Install MySQL and MySQL Workbanch
```
Comment (Alekearl) [inline README.md]:
```suggestion
2. Install MySQL and MySQL Workbench
```

Comment (Alekearl) [reply README.md]: One letter wrong

File: README.md (hunk 1 above)
Comment (Alekearl) [reply README.md]: Forgotten coma

**Review by Alekearl — CHANGES_REQUESTED** [review]
moments with written text

**Review by RomanDubovskyi — APPROVED** [review]
all good

**Pull request merged**